forms/FormField.php

```diff
@@ -1039,12 +1039,18 @@ public function setDisabled($disabled) {
 	 * @return FormField
 	 */
 	public function performReadonlyTransformation() {
-		$readonlyClassName = $this->class . '_Readonly';
+		// Strip suffix so the transformation works on already transformed classes
+		$class = preg_replace('/(_readonly|_disabled)$/i', '', $this->class, 1);
```

Comment:
I feel like it would be better that `ReadonlyField::performReadonlyTransformation()` should just do nothing instead.